Dockerfile

```diff
@@ -69,6 +69,7 @@ RUN apt-get --yes update \
   qemu-system-x86 \
   shellcheck \
   software-properties-common \
+  texinfo \
```

Comment:
Is this needed by newlib, or the configure_make Bazel rule?

Reply:
newlib itself, unfortunately. It wants to build documentation, and doesn't have a handy flag to disable that.